app/Resources/views/layout.html.twig

```diff
@@ -166,7 +166,7 @@
 <script type="text/javascript" language="javascript">
 $(function() {
 
-  initSearchBar('{{ path('uvweb_uv_nameslike') }}', '/unis/', '{{ path('uvweb_uv_detail') }}');
+  initSearchBar('{{ path('uvweb_uv_nameslike') }}', '{{ path('uvweb_foreignclass_details', { 'uvname': 'uvname', 'uniId': 'uniId' }) }}', '/unis/', '{{ path('uvweb_uv_detail') }}');
```

Comment:
It seems that function is expecting only 3 parameters: https://github.com/dotpy3/UVweb/blob/c23765ea11e624edb7773849757adbfc8408b12d/web/js/uvweb-js.js#L1